app/views/case_logs/edit.html.erb

```diff
@@ -3,6 +3,8 @@
 <%= turbo_frame_tag "case_log_form", target: "_top" do %>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds-from-desktop">
+      <% breadcrumbs = { "Logs" => "/logs", content_for(:title) => "" } %>
+      <%= govuk_breadcrumbs(breadcrumbs: breadcrumbs) %>
```

Comment:
The breadcrumbs should appear directly after the phase banner, resulting code looking something like this:

```html
<div class="govuk-width-container">
    <div class="govuk-phase-banner">…</div>

    <div class="govuk-breadcrumbs">…</div>

    <main class="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content" role="main">…</main>
</div>
```